packages/react-lowcode/src/codegen/tests/list/files/page-list-template.txt

```diff
@@ -1,4 +1,4 @@
-import { useGeneratedQuery } from '../generated'
+import useGeneratedQuery from '../generated'
```

Comment:
we should generate named imports
test your generated code in demo.iteria.app (use *.txt manully in the vscode web)